dcrlibwallet.go

```diff
@@ -152,6 +154,32 @@ func (lw *LibWallet) Shutdown(exit bool) {
 	}
 }
 
+func (lw *LibWallet) DeleteWallet(privatePassphrase []byte) error {
+
+	defer func() {
+		for i := range privatePassphrase {
+			privatePassphrase[i] = 0
+		}
+	}()
+
+	lw.CancelSync()
```

Comment:
Is this needed here? `lw.Shutdown` called in line 178 below also calls `lw.CancelSync()`.